.github/workflows/test.yml

```diff
@@ -1277,7 +944,6 @@ jobs:
           subjob ${{needs.turbopack_rust_test1.result}} "Turbopack Rust tests (linux)"
           subjob ${{needs.turbopack_typescript.result}} "Turbopack Typescript checks"
           subjob ${{needs.turborepo_rust_test.result}} "Turborepo Rust tests"
-          subjob ${{needs.turbopack_rust_test_bench1.result}} "Turbopack Rust benchmark tests (linux)"
```

Comment:
this makes it no longer required

Reply:
Isn't it still working the same way from the new workflow?  https://github.com/vercel/turbo/pull/6108/files#diff-9f350e5932ba25c06a424509fea40f34c62ac56cdbff4be5d8196202a99e497bR514-R515

Reply:
oh, I guess, but we need to change the repo settings and this will create a second comment for the summary for each PR

Reply:
> but we need to change the repo settings

I don't understand, can you clarify? Also I am not sure if your comment means this PR is breaking something or if it's a suggestion. Feel free to ❌ request changes if it's the former!

> this will create a second comment

yes I called this out in the PR description already. Do you consider this a good reason not to split?

> maybe instead of making it completely separate we can call it from this workflow like the cleanup job is called

I'm not opposed to this, but I've moved over 5 benchmarking jobs. Should all 5 of them be called from test.yml in your opinion? I feel like it's better to just be a separate workflow (also note that turborepo also has `bench-turborepo.yml`, so it would align well with that)